src/elm/Page/Shop.elm

```diff
@@ -553,6 +553,36 @@ viewShopFilterDropdown translations maybeFilter loggedIn =
     viewMenuFilterDropdown ClickedFilter decoder options
 
 
+viewShopFilterTab :  ( String, String, String ) -> Maybe Filter -> LoggedIn.Model -> Html Msg
```

Comment:
@JerryNyoike  how come a Maybe Filter, if its always tabs shouldn't a filter always be provided, How could it end up being a Nothing?

Reply:
Oh, I see. I was looking at it from the point of view that only one filter could be applied at a time.I will try it this way instead